### PR TITLE
Added Dutch translation

### PR DESCRIPTION
**Pull request by sebastianschauenburg**

### CODE REVIEW

**Comment by sebastianschauenburg** [comment]
All local yarn tests passed. Unsure why the Github tests are failing.

**Comment by pavlobu** [comment]
Hey @sebastianschauenburg I created `fix-dutch` branch and it has passed checks. I can rebase your pull request branch or we can move to work on this https://github.com/pavlobu/deskreen/pull/114 pull request

**Comment by pavlobu** [comment]
Great work thanks!🙏 I'll be updating you with a website template here soon

**Comment by sebastianschauenburg** [comment]
@pavlobu thanks for pointing the error out. Clear oversight on my part, thank you :-)

**Review by lucalves — APPROVED** [review]

File: app/locales/nl/translation.json [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+  "Type a New Password": "Typ een Nieuw Wachtwoord",
+  "Cancel": "Annuleren",
+  "Device Status": "Apparaat Status",
+  "Sharing Screen": "Scherm Gedeeld",
+  "Available, no screen sharing": "Beschikbaar, geen scherm gedeeld",
+  "Not Available": "Niet Beschikbaar",
+  "Autostart Deskreen App on login": "Autostart Deskreen App tijdens inloggen",
+  "Open Deskreen App window on login": "Open Deskreen App venster tijdens inloggen",
+  "Use system tray": "Gebruik systeemvak",
+  "Deskreen System Tray": "Deskreen System Vak",
```
Comment (WikiRik) [inline app/locales/nl/translation.json:138]: Systeemvak*, in line with line 137

File: app/locales/nl/translation.json [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+  "All devices are successfully disconnected": "Alle apparaten hebben succesvol de verbinding verbroken",
+  "Device was disconnected": "Verbinding met apparaat was verbroken",
+  "Networking": "Netwerk",
+  "Deskreen Application Port": "Deskreen Applicatie Poort",
+  "Port is already used by other App": "Poort wordt is al in gebruik door een andere App",
+  "Click to change Deskreen Application Port": "Klik om Deskreen Applicatie Poort aan te passen",
+  "Type a number from 3000 to 64000 to use as a Deskreen Application Port": "Type een nummer van 3000 tot 64000 om te gebruiken als Deskreen Applicatie Poort",
+  "Type another number in range from 3000 to 64000": "Type een ander nummer in de reeks van 3000 tot 64000",
+  "Select Network Interface": "Selecteer Netwerk Interface",
+  "I know IP of my computer and I want to type it manually": "Ik weet het IP van mijn computer en wil het handmatig invoeren",
```
Comment (WikiRik) [inline app/locales/nl/translation.json:118]: intypen?, to be in line with lines 119 and 120

File: app/locales/nl/translation.json [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+  "Click to edit Device Alias": "Klik voor het wijzigen van het Apparaat Alias",
+  "Trusted Device ID": "Vertrouwd Apparaat ID",
+  "Trusted": "Vertrouwd",
+  "Make Trusted": "Maak Vertrouwd",
+  "Forget This Device": "Vergeet Dit Apparaat",
+  "Device Alias": "Apparaat Alias",
+  "Auto share last Entire Screen source when device is available": "Deel automatisch de laatste scherm-bron wanneer het apparaat beschikbaar is",
+  "All devices are successfully disconnected": "Alle apparaten hebben succesvol de verbinding verbroken",
+  "Device was disconnected": "Verbinding met apparaat was verbroken",
+  "Networking": "Netwerk",
```
Comment (WikiRik) [inline app/locales/nl/translation.json:111]: Not sure where this line is used, but maybe make it 'Netwerken'?

File: app/locales/nl/translation.json [hunk 4]
```diff
@@ -0,0 +1,143 @@
...
+  "Color Theme": "Kleur Thema",
+  "Automatic Updates": "Automatische Updates",
+  "General Settings": "Algemene Instellingen",
+  "Disabled": "Uitgeschakeld",
+  "Version": "Versie",
+  "Copyright": "Copyright",
+  "Edit": "Wijzig",
+  "Hide Deskreen": "Verberg Deskreen",
+  "Hide Others": "Verberg Andere",
+  "Show All": "Toon alle",
```
Comment (WikiRik) [inline app/locales/nl/translation.json:77]: Toon Alle*

File: app/locales/nl/translation.json [hunk 5]
```diff
@@ -0,0 +1,143 @@
...
+  "View": "Bekijken",
+  "Reload": "Herladen",
+  "Toggle Full Screen": "Schakelen Volledig Scherm",
+  "Toggle Developer Tools": "Schakelen Ontwikkelaars Gereedschappen",
+  "Window": "Scherm",
+  "Minimize": "Minimaliseer",
+  "Close": "Afsluiten",
+  "Bring All to Front": "Breng Alle naar Voren",
+  "Help": "Help",
+  "Learn More": "Leer meer",
```
Comment (WikiRik) [inline app/locales/nl/translation.json:94]: Leer Meer*

File: app/locales/nl/translation.json [hunk 6]
```diff
@@ -0,0 +1,143 @@
...
+  "Website": "Website",
+  "About Deskreen": "Over Deskreen",
+  "Security Settings": "Beveiligings instellingen",
+  "Color Theme": "Kleur Thema",
+  "Automatic Updates": "Automatische Updates",
+  "General Settings": "Algemene Instellingen",
+  "Disabled": "Uitgeschakeld",
+  "Version": "Versie",
+  "Copyright": "Copyright",
+  "Edit": "Wijzig",
```
Comment (WikiRik) [inline app/locales/nl/translation.json:74]: Maybe use 'Aanpassen'?

File: app/locales/nl/translation.json [hunk 7]
```diff
@@ -0,0 +1,143 @@
...
+  "You will have to connect all devices manually again": "U zult met alle apparaten handmatig opnieuw verbinding moeten maken",
+  "No, Cancel": "Nee, Annuleren",
+  "Yes, Disconnect All": "Ja, alle verbindingen verbreken",
+  "A new version of Deskreen is available! Click to download new version": "Een nieuwe versie van Deskreen is beschikbaar. Klik om de nieuwe versie te downloaden!",
+  "Security": "Beveiliging",
+  "General": "Algemeen",
+  "About": "Over",
+  "Website": "Website",
+  "About Deskreen": "Over Deskreen",
+  "Security Settings": "Beveiligings instellingen",
```
Comment (WikiRik) [inline app/locales/nl/translation.json:67]: Beveiligingsinstellingen*

File: app/locales/nl/translation.json [hunk 8]
```diff
@@ -0,0 +1,143 @@
...
+  "Application Window": "Applicatie Venster",
+  "Check if all is OK and click Confirm": "Controleer of alles OK is en klik Bevestigen",
+  "Confirm Button Text": "Bevestigen",
+  "No, I need to choose other": "Nee, ik moet iets anders kiezen",
+  "Done!": "Gereed!",
+  "Now you can see your screen on other device": "U kunt nu uw scherm zien op het andere apparaat",
+  "You can manage connected devices by clicking Connected Devices button in top panel": "U kunt verbonden apparaten beheren door op Verbonden Apparaten te klikken in het bovenste paneel",
+  "Connect New Device": "Nieuw Apparaat Verbinden",
+  "Select Entire Screen to Share": "Selecteer gehele scherm om te delen",
+  "Select App Window to Share": "Selecteer applicatie venster om te delen",
```
Comment (WikiRik) [inline app/locales/nl/translation.json:53]: App venster? In line with the other translations of App

File: app/locales/nl/translation.json [hunk 9]
```diff
@@ -0,0 +1,143 @@
...
+  "Connected": "Verbonden",
+  "Click to see more": "Klik om meer te zien",
+  "This should match with Device IP displayed on the screen of device that is trying to connect": "Dit moet identiek zijn aan het Apparaat IP welke weergegeven is op het scherm van het apparaat dat probeert verbinding te maken",
+  "If IP addresses dont match click Disconnect button": "Als de IP adressen niet overeenkomen klik de Verbinding verbreken knop",
+  "Disconnect": "Verbinding verbreken",
+  "Choose Entire Screen or App window you want to share": "Kies het gehele scherm of het applicatie venster dat u wilt delen",
+  "OR": "OF",
+  "Entire Screen": "Gehele Scherm",
+  "Application Window": "Applicatie Venster",
+  "Check if all is OK and click Confirm": "Controleer of alles OK is en klik Bevestigen",
```
Comment (WikiRik) [inline app/locales/nl/translation.json:45]: 'alles goed' might be better

File: app/locales/nl/translation.json [hunk 10]
```diff
@@ -0,0 +1,143 @@
...
+  "Device is successfully disconnected by you You can connect a new device": "Apparaat heeft succesvol de verbinding verbroken U kunt een nieuw apparaat verbinden",
+  "Deskreen Update is Available!": "Deskreen Update is Beschikbaar!",
+  "Your current version is": "Uw huidige versie is",
+  "Click to download new updated version": "Klik om de nieuwe geüpdate versie te downloaden",
+  "Connected": "Verbonden",
+  "Click to see more": "Klik om meer te zien",
+  "This should match with Device IP displayed on the screen of device that is trying to connect": "Dit moet identiek zijn aan het Apparaat IP welke weergegeven is op het scherm van het apparaat dat probeert verbinding te maken",
+  "If IP addresses dont match click Disconnect button": "Als de IP adressen niet overeenkomen klik de Verbinding verbreken knop",
+  "Disconnect": "Verbinding verbreken",
+  "Choose Entire Screen or App window you want to share": "Kies het gehele scherm of het applicatie venster dat u wilt delen",
```
Comment (WikiRik) [inline app/locales/nl/translation.json:41]: App instead of applicatie? In line with the other translations of App

**Review by WikiRik — COMMENTED** [review]
Just came across this PR and thought to go ahead and leave some comments. Good work overall!

**Comment by pavlobu** [comment]
Hi @sebastianschauenburg  @WikiRik @lucalves 

I finally finished language templates for Deskreen website.

Here are the guides on how to add a new language for Deskreen Website
https://github.com/Deskreen/deskreen-website-locales

Thank you in advance!
Paul

**Comment by pavlobu** [comment]
Still no Dutch translations for a website, so this pull request is stalled for now

**Comment by pavlobu** [comment]
and we have a merge conflict now also, this should be resolved

**Comment by pavlobu** [comment]
merged these changes in this pr
https://github.com/pavlobu/deskreen/pull/198

for website I used google translate, please review on your free time
https://github.com/Deskreen/deskreen-website-locales/tree/main/nl

thanks

**Pull request closed without merging**